blocks/init/src/Blocks/custom/tabs/manifest.json

```diff
@@ -0,0 +1,90 @@
+{
+	"blockName": "tabs",
+	"title": "Tabs",
+	"description" : "Simple tabs block",
+	"category": "eightshift",
+	"icon": {
+		"src": "es-example"
+	},
+	"keywords": [
+		"tabs"
+	],
+	"example": {
+		"attributes": {
+			"tabsCurrentlyActiveTabId": "exampleTab1"
+		},
+		"innerBlocks": [
+			{
+				"name": "eightshift-boilerplate/tab",
+				"attributes": {
+					"tabTitle": "Example tab 1",
+					"tabId": "exampleTab1"
+				},
+				"innerBlocks": [
+					{
+						"name": "eightshift-boilerplate/card",
+						"attributes": {
+							"imageUrl": "https://picsum.photos/400/400",
+							"introContent": "Intro",
+							"introSize": "small",
+							"introColor": "light",
+							"headingContent": "Heading",
+							"paragraphContent": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam aliquam est id semper aliquet.",
+							"buttonUse": false
+						}
+					}
+				]
+			},
+			{
+				"name": "eightshift-boilerplate/tab",
+				"attributes": {
+					"tabTitle": "Example tab 2",
+					"tabId": "exampleTab2"
+				},
+				"innerBlocks": [
+					{
+						"name": "eightshift-boilerplate/card",
+						"attributes": {
+							"imageUrl": "https://picsum.photos/400/400",
+							"introContent": "Intro",
+							"introSize": "small",
+							"introColor": "light",
+							"headingContent": "Heading",
+							"paragraphContent": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam aliquam est id semper aliquet.",
+							"buttonUse": false
+						}
+					}
+				]
+			}
+		]
+	},
+	"hasInnerBlocks": true,
+	"attributes": {
+		"tabsAllowedBlocks": {
+			"type": "array",
+			"items": {
+				"type": "string"
+			},
+			"default": [
+				"eightshift-boilerplate/tab"
+			]
+		},
+		"tabsCurrentlyActiveTabId": {
+			"type": "string"
+		}
+	},
+	"providesContext":{
+		"eightshift-boilerplate/tabsCurrentlyActiveTabId": "tabsCurrentlyActiveTabId"
+	},
+	"blockClass": "block-tabs",
```

Comment:
you have all of this in the $attributes. 

and in general we dont do this.
        "blockJsClass": you have in attributes
	"tabJsClass": you have in attributes
	"tabPanelElement": all selectors must be prefixed with JS
	"tabPanelItemElement": all selectors must be prefixed with JS
	"tabPanelItemButtonElement": all selectors must be prefixed with JS
	"tabPanelJsClass": look after for the example
	"tabVisibleClass": use simple selectors like: is-vissible, is-active, etc
	"tabPanelItemButtonJsClass": look after for the example
	"tabPanelItemActiveClass": use simple selectors like: is-vissible, is-active, etc
	"tabPanelItemTemplateId": 

All Js selectors must be prefixed with js-block-name.  In you class you only need to pass blockJsClass and bind all selectors on that like `.${blockJsClass}-panel` or `.${blockJsClass}-button`, etc.

But we like to give options to configure the JS as much as possible so you can pass all of this selectors in your class like this. https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/components/accordion/assets/index.js